src/content/changelog/4-74-0-2020-09-08.md

```diff
@@ -0,0 +1,27 @@
+---
+title: '4.72.0: 2020-09-08'
+date: 2020-09-08T04:00:00.000Z
+version: 4.74.0
+changelog:
+  - API
+---
+
+### Added
+
+* Added the *Reset Linode Root Password* ([POST /linode/instances/{linodeId}/password](/api/v4/linode-instances-linode-id-password/#post)) endpoint to the *Linode Instances* collection. Use this endpoint to reset the root password for a Linode.
+
+* Added the `bucket_access` field to the *Create an Object Storage Key* ([POST /object-storage/keys](/api/v4/object-storage-keys/#post)) endpoint. This field defines an Object Storage key as a *Limited Access Key*. Limited Access Keys have access to only the bucket(s) that are specified along with `read_write` or `read_only` bucket-level permissions. Limited Access Keys can:
```

Comment:
Should check with Vicki to make sure this is correct.

Reply:
Looks good to me! 👍